### DISCUSSION TITLE
Communication over websockets

### DISCUSSION THREAD

**Discussion by rithask** [RavenSystem/esp-homekit-devices#2028] [New Features]
Is it possible to communicate over websockets? My LG TV runs on WebOS and i want to control it using homekit.

**Comment by ghost** [comment]
I use homebridge with the LG WebOS Tv plug-in. Works a treat!

**Comment by rithask** [comment]
Currently, that's my setup. But I wanna try controlling the TV using esp-homekit-devices becuase I'll probably retire my rpi soon or use it for something else.

**Reply by ghost** [reply]
Interesting, that’s a great idea! I never thought of that approach. I’ll be watching this thread with interest 😊

**Reply by xrust83** [reply]
The webos plugin asks for a pairing code in the homebridge, I think it will be a hindrance
